src/_guides/language/effective-dart/usage_migrated.md

```diff
@@ -1110,6 +1110,15 @@ class ProfileMark {
 }
 {% endprettify %}
 
+One common reason that fields are *not* initialized at the declaration is
+because they need to reference `this` or call methods on the containing
+instance. However, if the field is marked `late`, then the initializer *can*
+access `this`. In cases where the initialization doesn't depend on constructor
+parameters, you may be able to move the initialization to the field's
+declaration. That doesn't necessarily mean you *should* use `late` just to be
+able to initialize at the declaration. But if you want the field to be `late`,
+it gives you additional flexibility.
```

Comment:
I don't think I understand this.

Why else would someone want the field to be late?

Is an author going to read this an push back against a CR comment saying "make this field late final so we can move the initialization out of the constructor" with "You should only do that if you want the field to be `late` and I don't have another reason to want that"?

Reply:
My intent was to not come across as strongly encouraging users to use `late` unless they actually need it. But I guess you're probably right that in cases where this applies, they probably *do* want to use `late` so it's fine. Removed those sentences.